### PR TITLE
feat: login

### PR DESCRIPTION
**Pull request by TonNQ**
Implement login for admin

86epkwke7

### CODE REVIEW

File: src/components/Input/Input.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { InputHTMLAttributes } from 'react'
+import type { RegisterOptions, UseFormRegister } from 'react-hook-form'
+
+interface InputProps extends InputHTMLAttributes<HTMLInputElement> {
```
Comment (ces-anhlam) [inline src/components/Input/Input.tsx:4]:
```InputProps<T> ...
....
register?:UseFormRegister<T>```

File: src/components/Input/Input.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      <div className={classNameLabel}>{labelName}</div>
+      <div className='relative'>
+        <input type='email' placeholder='Enter your email' className={classNameInput} {...registerResult} {...rest} />
+        {IconComponent && (
+          <span className={classNameIcon}>
+            <IconComponent />
+          </span>
+        )}
+      </div>
+      <div className={classNameError}>{errorMessage}</div>
```
Comment (ces-anhlam) [inline src/components/Input/Input.tsx:44]: check errorMessage

File: src/components/Input/Input.tsx (hunk 1 above)
Comment (TonNQ) [reply src/components/Input/Input.tsx:4]: fixed

**Review by ces-anhlam — APPROVED** [review]

**Review by ces-khanhhoang — APPROVED** [review]

**Comment by ces-anhlam** [comment]
@TonNQ Please fix conflicts

**Pull request merged**